executor/executor.go

```diff
@@ -117,9 +117,9 @@ func doRunCommand(guid, path string, emitter screwdriver.Emitter, f *os.File, fR
 }
 
 // Executes teardown commands
-func doRunTeardownCommand(cmd screwdriver.CommandDef, emitter screwdriver.Emitter, env []string, path, shellBin string, exportFile string) (int, error) {
+func doRunTeardownCommand(cmd screwdriver.CommandDef, emitter screwdriver.Emitter, env string, path, shellBin string, baseEnvFile string, exportFile string) (int, error) {
```

Comment:
can omit `string` for a couple params

```suggestion
func doRunTeardownCommand(cmd screwdriver.CommandDef, emitter screwdriver.Emitter, env, path, shellBin, baseEnvFile, exportFile string) (int, error) {
```